### DISCUSSION TITLE
Welcome to nodel Discussions!

### DISCUSSION THREAD

**Discussion by paxofsnax** [museumsvictoria/nodel#293] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dargs** [comment]
Hello! I'm @dargs, an AV Tech @  ACMI. 

We've been running Nodel as our main control system since 2020.

@justparking & @mcartmel from Lumicom installed, setup and they remotely monitor our Nodel systems for our permanent Exhibition (Story of the Moving Image). 

They've been super supportive when we have questions and couldn't have done it without them! Side note - they were involved in the original ACMI digital signage and control system also - 20+ years ago !

We've since been adding Nodel to almost every other space in the building, to work with our systems for galleries and event/production AV Control & Monitoring.

Some stuff we have or is a work in progress - 

- Dynalite (100+ Lighting Areas, Blinds, data from Antumbra panels (temp / motion), enable/disabling panels & motion sensing during events using timers)

- Pharos LPC DMX Controllers  - Dynalite mostly controls timeline triggers but Nodel could too. 

- Yamaha DSPs, Mixers and Amps. Nodel control for Background music selection / foyer announcements, recall scenes on CL3/QL1 consoles)  

- QSC Q-Sys Core - Justin has integrated our Gallery q-sys control/metering 

- Spotify Control (Play/Pause, Playlist selection etc) - WIP 

- Panasonic Projectors, Screens, PTZ Cameras. Screen Technics motorised screens. 

- Shure ULXD Radio Mic receivers - monitor RF / Audio / Battery data ~30 channels of radio mics for events

- Ross video Ultrix Routing & Carbonite Switchers - recalls routing and controls switchers

- Blackmagic Design - controlling a Video Hub router (SDI Fibre routing around the building/Fed Sq)

- Brightsign players - Play/Pause/Mute etc. 

- ma Lighting - ma2 onPC control - Loads showfiles, triggers cues, blackouts etc. 

- Slack - sending slack messages from Nodel groups when devices go offline or thresholds reached..

- Ungerboeck (EBMS -  Event Booking Management System) - retrieving event bookings and functions 

- Scala Digital Signage (Monitoring players, triggering playlists for 'Record In Progress / On Air' signage, sending REST actions to nodel to change Audio DSP Presets etc)

- EWIS (Monitoring Fire Panel circuits to trigger audio mutes (and potentially digital signage / lighting / AV states in emergencies) via Global Cache iTach Flex IP GPIO monitoring. 

- Zoom Video conferencing (boardroom systems.. potential to also monitor meeting stats, stop/start cloud recording etc)  

- Outlook Calendars - expanding our usage to eventually become the sole AV scheduling system. (removing schedules from Dynalite/Creston / legacy systems)

- XOS - ACMI's 'Museum operating system' - nodel monitors Raspi & Dell Balena players and can get stats

- Cisco switches - Justin had port enable/disable control as way of powering on/off PoE devices (Raspi 'Lens Readers'), I've recently added more port status feedback events (description/vlan/bandwidth usage etc) 

- Jira / Confluence - tbc eventually integrate to create helpdesk tickets etc.

- Snipe-IT - Equipment Asset management - tbc eventually integrate 

- Nodel Text to Speech using Mac OSX 'Say' command, _giving nodel a voice_ :) .. bit of fun, plus also looking at connecting our Motorola 2-way radio & RTS Intercom system to audibly 'say' critical status updates over the radio to Technicians on duty. Or it might just say Hello now and again.. 

- Streamdeck & Companion - physical buttons to send REST Api actions, [Companion](https://github.com/bitfocus/companion) also has lots of device control API add-ons to borrow from for Nodel.

I'm still learning github basics so havent forked/pushed many of my code examples yet, one day I'll work it out! And would like to contribute to Nodel documentation, even just basic code snippets / examples to jog peoples memory when writing new functions. 

Look forward to catching up with more of the nodel users one day soon ! 

dargs
